consensus/polybft/state_sync_manager.go

```diff
@@ -602,3 +634,16 @@ func (s *stateSyncManager) multicast(msg interface{}) {
 		s.logger.Warn("failed to gossip bridge message", "err", err)
 	}
 }
+
+// parseStateSyncResultEvent parses StateSyncResult event from the provided log
+func parseStateSyncResultEvent(h *types.Header, l *ethgo.Log) (*contractsapi.StateSyncResultEvent, bool, error) {
+	var stateSyncResultEvent contractsapi.StateSyncResultEvent
+
+	if len(l.Topics) == 0 || l.Topics[0] != stateSyncResultEvent.Sig() {
```

Comment:
No need for this check. I would also remove this function completely, and just add this parse code inline where you instantiate `eventGetter`.
```suggestion
	var stateSyncResultEvent contractsapi.StateSyncResultEvent
	doesMatch, err := stateSyncResultEvent.ParseLog(l)

	return &stateSyncResultEvent, doesMatch, err
```

Reply:
fixed: 118901573a263bb979712ae7984a8c22d6246db5